### PR TITLE
Update of making cleaner code and organized files

### PR DESCRIPTION
**Pull request by Ljanjic**
Based on Vitali's feedback, did a better organization to improve the readability of our back-end code, by destructuring code in controllers and routes files, creating Auth(Register, Login, Logout), User(edit User Profile, Delete User account), and Activity (routes)files

### CODE REVIEW

File: src/controllers/activityController.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ const getActivity = async (req, res) => {
     }
     res.status(StatusCodes.OK).json({ activity });
   } catch (error) {
-    console.error("Error in getActivity:", error);
+    console.error('Error in getActivity:', error);
```
Comment (vitaliipp) [inline src/controllers/activityController.js:48]: Please remove lines:
36 user: { userId },
41 createdBy: userId,
getActivity should be unprotected

File: src/app.js [hunk 2]
```diff
@@ -34,11 +30,9 @@ app.use(
...
 
 // ROUTES
-app.use("/api/v1/auth", authRouter);
-app.use("/api/v1/activities", allActivitiesRouter);
-app.use("/api/v1/myActivities", authenticateUser, myActivitiesRouter);
-
-app.use("/api/v1/auth/users", authenticateUser, authRouter);
+app.use('/api/v1/auth', authRouter);
+app.use('/api/v1/activities', activityRouter);
+app.use('/api/v1/auth/users', userRouter);
```
Comment (vitaliipp) [inline src/app.js:35]: Should be app.use('/api/v1/users', userRouter);

**Review by vitaliipp — CHANGES_REQUESTED** [review]
Great job overall. Please split pull requests into small parts as much as you can. Don't forget to pull the main branch to avoid merging conflicts.

**Review by vitaliipp — APPROVED** [review]
Looks good to me

**Review by benochi — APPROVED** [review]

**Pull request merged**